network/ipcfg/ip.go

```diff
@@ -39,6 +40,11 @@ func SetIPv4ProxyARP(ifName string, value int) error {
 	return set(fmt.Sprintf(ipv4ProxyARP, ifName), value)
 }
 
+// SetIPv4ProxyDelay sets the IPv4 delay before responding to proxy ARP
```

Comment:
Can we please rename this function to "SetIPv4ProxyARPDelay"? I know the proc filename doesn't have "ARP" but the function would be more intuitive if we say this is the delay for ARP Proxy.
Comment could be "SetIPv4ProxyARPDelay sets the IPv4 proxy ARP response delay property of an interface to the given value."